Sources/Boutique/Store.swift

```diff
@@ -85,39 +74,32 @@ public final class Store<Item: Codable & Equatable>: ObservableObject {
         for item in uniqueItems {
             if let matchingIdentifierIndex = itemKeys.firstIndex(of: item[keyPath: self.cacheIdentifier]),
                case let matchingIdentifier = itemKeys[matchingIdentifierIndex],
```

Comment:
These two can be expressed in one: `if let matchingIdentifier = itemKeys.first(where: { $0 == item[keyPath: self.cacheIdentifier] }),`